pkg/controllers/provisioning/batcher.go

```diff
@@ -59,26 +62,24 @@ func (b *Batcher) TriggerImmediate() {
 // Wait starts a batching window and continues waiting as long as it continues receiving triggers within
 // the idleDuration, up to the maxDuration
 func (b *Batcher) Wait(ctx context.Context) {
-	var start time.Time
```

Comment:
Do you know why `start` was defined here?  Seems like it has no impact to remove, but I'm not sure if I'm missing something.

Reply:
It was originally used because a window was returned from the func a while back. This is dead code now. If we need it later, we can add it back later.

Reply:
gotcha, that's what I expected.